internal/cmd/cmd_zrange_watch.go

```diff
@@ -73,7 +73,7 @@ func evalZRANGEWATCH(c *Cmd, s *dstore.Store) (*CmdRes, error) {
 }
 
 func executeZRANGEWATCH(c *Cmd, sm *shardmanager.ShardManager) (*CmdRes, error) {
-	if len(c.C.Args) == 0 {
```

Comment:
`==0` is correct. For `.watch` commands the check on arguments is handled at primary command level. This way, we need not make changes at two places when we support other args. Revert this change.